Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -720,6 +726,18 @@ public final class PackageBuilder {
         }
     }
 
+    /// Validates module alias key and value pairs and throws an error if empty, is a stdlib, or
+    /// contains invalid characters.
+    private func validateModuleAliases(_ aliases: [String: String]?) throws {
+        guard let aliases = aliases else { return }
+        for (entKey, entVal) in aliases {
+            if !entKey.isValidIdentifier || entKey == "Swift" ||
```

Comment:
can you elaborate on the `== "Swift" condition? not clear to me what / how this is used

Reply:
It's to check against a stdlib name, but since that check is done in the frontend, will remove it here